evaluations/filecoin.md

```diff
@@ -26,8 +26,6 @@ Dagger. It is not meant to be a criticism of Filecoin.
 - Filecoin requires its own very specific blockchain, which influences a lot
   of its design. There is tight coupling between the blockchain, storage
   accounting, proofs and markets.
-- Proof of spacetime is much more complex than simple challenges, and only
```

Comment:
I think a better wording would be that "proofs of space time are (by design) computationally intensive and non-parallelizable (afaik)"